web/app/scripts/navbar/module.js

```diff
@@ -2,6 +2,9 @@
     'use strict';
 
     angular.module('driver.navbar', [
+        'driver.resources.recordstate',
+        'driver.resources.polygonstate',
+        'driver.resources.geographystate',
```

Comment:
It might be overkill to have all of these in separate modules. Maybe something like `driver.state` or `driver.storage` would be better?